### PR TITLE
chore(ui) Remove old icons

### PR DESCRIPTION
**Pull request by markstory**
* Remove the sentry warnings I added to track usage of deprecated icons.
* Remove unused icon files.
* Fix size of icon in settings > repositories as it was too small.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/alert.tsx [hunk 1]
```diff
@@ -92,27 +89,14 @@ const Alert = styled(
...
     ...props
   }: AlertProps) => {
-    if (typeof icon === 'string') {
-      withScope(scope => {
-        scope.setLevel(Severity.Warning);
-        scope.setTag('icon', icon);
-        scope.setTag('componentType', 'alert');
-        captureException(new Error('Deprecated SVG icon referenced'));
-      });
-    }
```
Comment (scttcper) [inline src/sentry/static/sentry/app/components/alert.tsx:108]: nice cleanup

**Review by scttcper — APPROVED** [review]
🎉 nice

**Review by billyvg — CHANGES_REQUESTED** [review]
I believe `icon-lock` is still in use by a plugin in `getsentry`

**Comment by markstory** [comment]
@billyvg You're right. The youtrack plugin still references the icon-lock and icon-checkmark CSS classnames. I missed those when I was removing css class names. I've added those classnames back which will fix the youtrack plugin.

**Review by billyvg — APPROVED** [review]

**Pull request merged**